ansible_base/rbac/api/serializers.py

```
@@ -110,6 +110,9 @@ def get_by_ansible_id(self, ansible_id, requesting_user, for_field):
             if obj._meta.model_name == 'user':
                 if not visible_users(requesting_user).filter(pk=obj.pk).exists():
                     raise ObjectDoesNotExist
+            elif obj._meta.model_name == 'team':
+                if not visible_teams(requesting_user).filter(pk=obj.pk).exists():
+                    raise ObjectDoesNotExist
```

Ruff (0.15.2)

[warning] Abstract `raise` to an inner function

(TRY301)